Update Jewish calendar omer counting service documentation (#38706)
home-assistant/home-assistant.github.io: source/_integrations/jewish_calendar.markdown

| Data attribute | Optional | Description                              |
| -------------- | -------- | ---------------------------------------- |
| `date`         | yes    | Date for which to get the Omer blessing. Defaults today|

> | `is_after_sunset`| yes  | if true and a date is provided, calculates the Omer count for the next day. Ignored if no date is specified.|

Reviewer: ```suggestion
| `after_sunset`| yes  | If true and a date is provided, calculates the Omer count based on the Hebrew date which starts after sunset. Ignored if no date is specified.|
```

> | `date`         | yes    | Date for which to get the Omer blessing. Defaults today|
> | `after_sunset`| yes  | If true and a date is provided, calculates the Omer count based on the Hebrew date which starts after sunset. Ignored if no date is specified.|
> | `nusach`       | no     | Nusach (tradition) of the Omer blessing. |
> | `language`     | yes    | Language to return. Defaults to Hebrew.  |

Reviewer: Coderabbit AI suggestions

```suggestion
| `date`         | yes    | Date for which to get the Omer blessing. Defaults to today. |
| `after_sunset` | yes  | If true and a date is provided, calculates the Omer count based on the Hebrew date which starts after sunset. Ignored if no date is specified. Defaults to True |
| `nusach`       | no     | Nusach (tradition) of the Omer blessing. |
| `language`     | yes    | Language to return. Defaults to Hebrew. |
```

action: jewish_calendar.count_omer
data:
  nusach: sfarad

>    data:
>      nusach: sfarad
>      date: "2025-05-20"       # optional; defaults to today
>      language: en             # optional; defaults to Hebrew
>      after_sunset: true       # optional; defaults to true

Reviewer: ```suggestion
  date: "2025-05-20"       # optional; defaults to today
  language: en             # optional; defaults to Hebrew
  after_sunset: true       # optional; defaults to true
```

data:
  nusach: sfarad
  language: en

>   is_after_sunset: true

Reviewer: ```suggestion
  after_sunset: true
```

days: 2
total_days: 37
```

> #### Minimal  call

Reviewer: ```suggestion
#### Minimal call
```

```yaml
action: jewish_calendar.count_omer
data:
  nusach: sfarad
```

> Will return the current text in Hebrew based on current Hebrew date (before/after  sunset)

Reviewer: ```suggestion
Will return the current text in Hebrew based on the current Hebrew date (taking into consideration the current time w.r.t. sunset)
```

> #### Minimal call
> ```yaml
> action: jewish_calendar.count_omer
> data:
>   nusach: sfarad
> ```
> Will return the current text in Hebrew based on the current Hebrew date (taking into consideration the current time w.r.t. sunset)

Reviewer: Add spacing (unable to create suggestion correctly due to ```)

```suggestion

#### Minimal call

```yaml
action: jewish_calendar.count_omer
data:
  nusach: sfarad
```

Will return the current text in Hebrew based on the current Hebrew date (taking into consideration the current time w.r.t. sunset)
```

days: 2
total_days: 37
```

#### Minimal call

```yaml
action: jewish_calendar.count_omer
data:
  nusach: sfarad
```

> Will return the current text in Hebrew based on the current Hebrew date (taking into consideration the current time w.r.t. sunset)

Reviewer: I didn't mean to delete this, I only wanted to add a whitespace between ``` and the text.
